config/types/node.go

```diff
@@ -54,3 +55,24 @@ func (n Node) Depth() int {
 	}
 	return count
 }
+
+func (nu NodeUser) Validate() report.Report {
+	r := report.Report{}
+	if nu.ID != 0 && nu.Name != "" {
```

Comment:
I just made the `id` fields in user and group null-able, so it can catch that now